src/aggregator/client/queue.go

```diff
@@ -204,22 +209,61 @@ func (q *queue) Close() error {
 	return nil
 }
 
+func (q *queue) writeAndReset() {
+	if len(q.buf) == 0 {
+		return
+	}
+	if err := q.writeFn(q.buf); err != nil {
+		q.log.Error("error writing data on timed flush",
+			zap.String("instance", q.instance.Endpoint()),
```

Comment:
Can you make this "target instance" so just thats its clear its the host you're trying to write to, not from